lib/src/components/NxModal/__tests__/NxModal.test.tsx

```diff
@@ -20,56 +20,59 @@ describe('NxModal', function() {
     onClose: dummyCloseHandler
   };
 
-  const getShallow = getShallowComponent<Props>(NxModal, minimalProps),
-      getModal = (props?: Partial<Props>) => getShallow(props).children();
+  const getMounted = getMountedComponent<Props>(NxModal, minimalProps);
 
   it('renders a context provider around an nx-modal-backdrop <dialog> containing an nx-modal <div>', function () {
```

Comment:
This test's name no longer matches what it checks

Reply:
> This test's name no longer matches what it checks

Fixed: 10f236378a8d638f74cc773d8d722c3276b13dad